Apps/Playground/CMakeLists.txt

```diff
@@ -173,6 +174,8 @@ if(APPLE)
             FOLDER "Playground")
     endif()
     set_property(TARGET Playground PROPERTY UNITY_BUILD false)
+    set_property(TARGET Playground PROPERTY XCODE_GENERATE_SCHEME YES)
+    set_property(TARGET Playground PROPERTY XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED YES)
```

Comment:
This change is an additional fix required by the bump of `ios-cmake`. 

More details: https://github.com/leetal/ios-cmake/pull/208#issuecomment-2235947477